apps/demo/config/components/Header/index.tsx

```diff
@@ -15,8 +15,8 @@ const NavItem = ({ label, href }: { label: string; href: string }) => {
       style={{
```

Comment:
Would be nice to get the actions within the header on the tab index - what's stopping that from working?

Reply:
Browser setting, you need to enable "tab to links" or similar.

Reply:
D'oh, was testing in Safari, works in Chrome!